specification/maps/data-plane/Search/preview/2021-11-01-preview/geocoding.json

```diff
@@ -0,0 +1,686 @@
+        "operationId": "GetGeocoding",
+        "produces": [
+          "application/geo+json"
+        ],
+        "x-ms-examples": {
+          "Search detail address 15127 NE 24th Street, Redmond, WA 98052": {
+            "$ref": "./examples/ForwardGeocoding.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          },
+          {
+            "$ref": "#/parameters/Top"
+          },
+          {
+            "name": "addressLine",
+            "in": "query",
+            "description": "The official street line of an address relative to the area, as specified by the locality, or postalCode, properties. Typical use of this element would be to provide a street address or any official address.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/CountryRegionSet"
+          },
+          {
+            "$ref": "#/parameters/Bbox"
+          },
+          {
+            "$ref": "#/parameters/View"
+          },
+          {
+            "$ref": "#/parameters/Location"
+          },
+          {
+            "name": "adminDistrict",
+            "in": "query",
+            "description": "The country subdivision portion of an address, such as WA.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "adminDistrict2",
+            "in": "query",
+            "description": "The county for the structured address, such as King.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "adminDistrict3",
+            "in": "query",
+            "description": "The named area for the structured address.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "locality",
+            "in": "query",
+            "description": "The locality portion of an address, such as Seattle.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "postalCode",
+            "in": "query",
+            "description": "The postal code portion of an address.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "strictMatch",
+            "in": "query",
+            "description": "Restrict the geocoding result to the country or region that is specified in the countryRegion field and the state, province or territory specified in the adminDistrict field.",
```

Comment:
There is no countryRegion field, just countryRegionSet.

Reply:
updated